### DISCUSSION TITLE
How to ensure we are 100% up to date?  (problems after last update)

### DISCUSSION THREAD

**Discussion by Aaron-E-Walsh** [mozilla/hubs#5239] [General Questions & Discussion]
Hello everyone, I wonder if there is a way to be certain that we are running the most current version of Hubs Cloud and client?  I ask because we did an update about a month or so ago, approximately, and since then we get broken link place holders every time a webpage is placed into a room.

I also notice a difference between the user interface that appears when we use Mozilla.com/hubs and what we see on our own AWS instance of Hubs Cloud / client.

So clearly we need to try and do the update again, but before we do that I wonder if there a way to tell what version of cloud and client we're running so that in the future we can know if we are out of date?

With thanks for your advice,
Aaron

**Comment by pattersonbl2** [comment]
Hubs.Mozilla.com and AWS hubs cloud have two different release cycles.  Some features may be released to hubs.mozilla may not be on AWS hubs and vice versa. Any package release for new code is updated automatically.   Cloudformation is done manually, this is where many miss updates. There isn't a way to tell if your cloud formation is on the latest.  You would have to go back to your Marketplace subscription and see if a new version is out.  We normally this would be an announcement in Discord.

**Reply by yengalvez** [reply]
Is there any way to have the most updated version of Mozilla Hubs on AWS? Maybe by doing something other than cloud formation? I'm also worried about paying for cloud formation to be versions behind. I need to keep applying improvements.

**Reply by pattersonbl2** [reply]
Cloud formation would be the best way to get the most updated currently. Feature updates are via the automatic updates built into hubs. Cloud formation only updates if  AWS resource requirements change, this doesn't happen often.

**Reply by yengalvez** [reply]
Oh! Now I understand. The updates for cloudformation of cloud hubs are twofold. One for cloudformation if it requires something from AWS. And then the Hubs Cloud itself is updated more periodically.

**Reply by Aaron-E-Walsh** [reply]
Thank you, Brandon. We have not spent as much time on Discord as on GitHub, thinking GitHub was the best place for updates and information. We will spend more time on Discord going forward, however.

We'll also use cloud formation as the main way to stay up-to-date. So far it has been working just fine, although the issue of all website thumbnails no longer appearing has been strange. So I think we'll update again and see how it goes.

With thanks again,
Aaron
